uvm/impl/com/untangle/uvm/MailSenderImpl.java

```diff
@@ -569,7 +578,7 @@ private void writeConfiguration()
              */
             String targetHost = null;
             if (settings.getSendMethod() == MailSettings.SendMethod.CUSTOM) targetHost = settings.getSmtpHost();
-            if (settings.getSendMethod() == MailSettings.SendMethod.RELAY) targetHost = "127.0.0.2"; // dot-two is NOT a typo
+            //if (settings.getSendMethod() == MailSettings.SendMethod.RELAY) targetHost = "127.0.0.2"; // dot-two is NOT a typo
```

Comment:
I think it will best for us to just leave the code there instead of commenting it.